playbooks/virt/openshift-cluster/launch_instances.yml

```diff
@@ -0,0 +1,102 @@
+- name: Create the libvirt storage directory for openshift
+  file:
+    dest: '{{ libvirt_storage_pool_path }}'
+    state: directory
+
+- name: Download Fedora Cloud image
+  get_url:
+    url: http://download.fedoraproject.org/pub/fedora/linux/releases/21/Cloud/Images/x86_64/Fedora-Cloud-Base-20141203-21.x86_64.qcow2
+    sha256sum: 3a99bb89f33e3d4ee826c8160053cdb8a72c80cd23350b776ce73cd244467d86
+    dest: '{{ libvirt_storage_pool_path }}/Fedora-Cloud-Base-20141203-21.x86_64.qcow2'
```

Comment:
Does CentOS also release “cloud images”? Either in qcow2 or even in raw format?

On the [CentOS download page](https://www.centos.org/download/), I could only find ISOs of LiveDVDs, Amazon’s AMIs and docker base image.
Nothing as convenient to use as a “cloud image”.